e2e/nomostest/config_sync.go

```diff
@@ -90,14 +90,22 @@ var (
 	hydrationPollingPeriod time.Duration
 )
 
-// IsReconcilerManagerConfigMap returns true if passed obj is the
+// IsReconcilerManagerCmConfigMap returns true if passed obj is the
 // reconciler-manager ConfigMap reconciler-manager-cm in config-management namespace.
-func IsReconcilerManagerConfigMap(obj client.Object) bool {
+func IsReconcilerManagerCmConfigMap(obj client.Object) bool {
```

Comment:
perhaps let's call this something like `IsReconcilerTemplateConfigMap` to reduce future confusion. WDYT?

Reply:
SGTM